rotkehlchen/tests/api/test_assets.py

```diff
@@ -306,7 +306,7 @@ def test_get_all_assets(rotkehlchen_api_server):
     assert 'entries_total' in result
     assert 'entries_limit' in result
     for entry in result['entries']:
-        assert 'Uniswap' in entry['name']
+        assert 'uniswap' in entry['name'].casefold()
```

Comment:
why did this change? Also we use.lower() in the codebase not casefold for this.

Reply:
Because some assets were added to the globaldb along with the pre-queried general cache. And some of those assets also match the uniswap query used in this test, however the case of the letters in the names of the new assets is different from what was expected here before.